datasets/tf_datasets.py

```diff
@@ -0,0 +1,117 @@
+"""
+https://www.tensorflow.org/datasets
+"""
+
+
+from typing import Optional
+from sisyphus import *
+
+
+class DownloadAndPrepareTfDatasetJob(Job):
+    """
+    https://www.tensorflow.org/datasets
+
+        pip install tensorflow-datasets
+
+    It further needs some extra dependencies, for example for 'librispeech':
+
+        pip install apache_beam
+        pip install pydub
+        # ffmpeg installed
+
+    See here for some more:
+    https://github.com/tensorflow/datasets/blob/master/setup.py
+
+    Also maybe::
+
+        pip install datasets  # for Huggingface community datasets
+    """
+
+    def __init__(
+        self,
+        dataset_name: str,
+        *,
+        max_simultaneous_downloads: Optional[int] = None,
+        max_workers: Optional[int] = None,
+    ):
+        """
+        :param dataset_name:
+            https://www.tensorflow.org/datasets/overview
+            https://www.tensorflow.org/datasets/catalog/overview
+            https://www.tensorflow.org/datasets/community_catalog/huggingface
+        :param max_simultaneous_downloads: https://github.com/tensorflow/datasets/issues/3885
+        :param max_workers:
+            For download extractor (https://github.com/tensorflow/datasets/issues/3887)
+            and Apache Beam.
```

Comment:
A link to a github issue is **not** a documentation. Please provide a documentation.

Reply:
```suggestion
        :param max_simultaneous_downloads: max simultaneous downloads
            (https://github.com/tensorflow/datasets/issues/3885)
        :param max_workers: max workers, for download extractor and Apache Beam
            (https://github.com/tensorflow/datasets/issues/3887)
```